packages/block-library/src/template-part/edit/placeholder.js

```diff
@@ -86,8 +85,7 @@ export default function TemplatePartPlaceholder( {
 
 			{ ! isResolving && isBlockBasedTheme && canCreateTemplatePart && (
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
This can be seen, while you add `core/template-part` and check for the placeholder button.

| Before | After     |
| :-------- | :------- |
| <img width="772" alt="template-part-before" src="https://github.com/user-attachments/assets/1d704184-22e0-4ec4-b58f-12e0911e109f"> | <img width="772" alt="template-part-after" src="https://github.com/user-attachments/assets/bc3948d7-c298-40db-b905-c2d78cefc781"> |